client/components/forms/FormContent.tsx

```diff
@@ -1,17 +1,19 @@
-import {ContactInfo} from '../../pages/contact-info';
+import { GeneralProjectInfo } from 'pages/general-project-info';
+import { ContactInfo } from '../../pages/contact-info';
```

Comment:
Importing from pages doesn't seem right, could these be defined in a constants folder?